gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/HttpBasicPassTicketScheme.java

```diff
@@ -80,13 +90,22 @@ public AuthenticationCommand createCommand(Authentication authentication, Suppli
 
         private static final long serialVersionUID = 3941300386857998443L;
 
+        private static final String COOKIE_HEADER = "cookie";
+
         private final String authorizationValue;
+        private final String cookieName;
         private final long expireAt;
 
         @Override
         public void apply(InstanceInfo instanceInfo) {
```

Comment:
You should handle the `public void applyToRequest(HttpRequest request)` method as well, not just the InstanceInfo variant

Reply:
You're right, it is new for me. Fixed

Reply:
Please add unit tests for this case